### PR TITLE
02 ilze mitre

### PR DESCRIPTION
**Pull request by zoyazip**

### CODE REVIEW

File: Majasdarbs/02-majasdarbs/Portfolio/ilze-mitre.html [hunk 1]
```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Ilze Mitre | Portfolio</title>
+  <meta name="description" content="" />
```
Comment (zoyazip) [inline Majasdarbs/02-majasdarbs/Portfolio/ilze-mitre.html]: Empty content

File: Majasdarbs/02-majasdarbs/Portfolio/ilze-mitre.html [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#contact-me">Contact me</a></li>
+      </ul>
+    </nav>
+  </header>
+
+
+  <!-- ABOUT ME -->
+  <section id="aboutme">
+    <img src="./IMG_ilze-mitre.jpeg" alt="" width="100px" height="130px" />
```
Comment (zoyazip) [inline Majasdarbs/02-majasdarbs/Portfolio/ilze-mitre.html]: Empty alt

**Review by zoyazip — COMMENTED** [review]
Checked

File: Majasdarbs/02-majasdarbs/Portfolio/ilze-mitre.html (hunk 1 above)
Comment (IlzeiMilzei) [reply Majasdarbs/02-majasdarbs/Portfolio/ilze-mitre.html]: fixed

File: Majasdarbs/02-majasdarbs/Portfolio/ilze-mitre.html (hunk 2 above)
Comment (IlzeiMilzei) [reply Majasdarbs/02-majasdarbs/Portfolio/ilze-mitre.html]: fixed

**Pull request closed without merging**